mods/cnc/languages/rules/en.ftl

```diff
@@ -539,3 +539,112 @@ actor-truck =
       Unarmed
    .name = Supply Truck
 
+
+## aircraft.yaml
+actor-tran-encyclopedia = The chinook is a flying transport capable of holding ten infantry. In general, it is used to transport commandos or engineers for backline-destruction.
+    \Like all aircraft, it can be handy in a pinch to crush a commando by landing! (alt)
+actor-heli-encyclopedia = The apache has more health than the orca and acts as a support unit. It works great with artillery, as it can kill any approaching light vehicles.
```

Comment:
Could the encyclopedia entries be merged into other actor translations like it is in d2k?

https://github.com/OpenRA/OpenRA/blob/150e28a67289324d6adf71207204bac74999d628/mods/d2k/languages/rules/en.ftl#L115-L136

Reply:
You have to do it manually.